react-app/src/features/faq/content/abpGuides.ts

```diff
@@ -0,0 +1,81 @@
+import abp1PDF from "@/assets/abp/IG_ABP1_AlternativeBenefitPlanPopulations.pdf";
+import abp2aPDF from "@/assets/abp/IG_ABP2a_VoluntaryBenefitPackageAssurances.pdf";
+import abp2bPDF from "@/assets/abp/IG_ABP2b_VoluntaryEnrollmentAssurances.pdf";
+import abp3PDF from "@/assets/abp/IG_ABP3_SelectionOfBenchmark20190819-Final.pdf";
+import abp31PDF from "@/assets/abp/IG_ABP3.1_SelectionOfBenchmarkAfter2020.pdf";
+import abp4PDF from "@/assets/abp/IG_ABP4_AbpCostSharing.pdf";
+import abp5PDF from "@/assets/abp/IG_ABP5_BenefitsDescription-Final.pdf";
+import abp6PDF from "@/assets/abp/IG_ABP6_BenchmarkEquivalentBenefit.pdf";
+import abp7PDF from "@/assets/abp/IG_ABP7_BenefitAssurances.pdf";
+import abp8PDF from "@/assets/abp/IG_ABP8_ServiceDeliverySystems.pdf";
+import abp9PDF from "@/assets/abp/IG_ABP9_EmployerSponsoredInsurance.pdf";
+import abp10PDF from "@/assets/abp/IG_ABP10_GeneralAssurances.pdf";
+import abp11PDF from "@/assets/abp/IG_ABP11_PaymentMethodology.pdf";
+
+export const abpGuides = [
```

Comment:
```suggestion
export const ABP_GUIDES: Template[] = [
```

Constants [should be uppercase](https://stackoverflow.com/questions/71029279/should-properties-of-constants-be-capitalized/71029393) for more context. Can you apply this change to all text constants? Oh, and please type each constant out with the `Template` type

Reply:
done.